README.md

```diff
@@ -6,6 +6,8 @@ CoreTools Healthcheck: [![CoreTools healthcheck](https://azure-functions.visuals
 
 ## V4 Images
 
+See release artifacts for Dockerfiles - example : https://github.com/Azure/azure-functions-docker/releases/download/3.4.2/3.4.2-appservice.zip
```

Comment:
@CooperLink - As of now artifacts only include `-appservice` images. Please open a separate WI to upload all the generated docker files